src/components/SimpleModal/SimpleModal.jsx

```diff
@@ -58,6 +62,29 @@ function SimpleModal({
     }
   }
 
+  function renderCloseButton() {
+    if (withCloseButton) {
+      if (customCloseButton && React.isValidElement(customCloseButton)) {
```

Comment:
That might be true, but I dislike allowing invalid states: `withCloseButton === false` should always turn it off, no matter any other prop. And since the defualt is true, you don't really have to wrry about it.